src/server/fs_extract.ts

```diff
@@ -227,16 +227,6 @@ export async function extractRoutes(
         name,
         component,
         handler: (req, ctx) => {
-          if (config.dev) {
```

Comment:
Oh no, I thought I had removed that part. I'm a bit mad at myself that I didn't catch that with a test. 🤦

Reply:
I know the feeling. I usually need 4 commits when changing CI stuff because I forget stupid things too 😅